packages/discord.js/src/structures/Presence.js

```diff
@@ -1,36 +1,41 @@
+/* eslint-disable no-use-before-define */
 'use strict';
 
-const { Base } = require('./Base.js');
-const { Emoji } = require('./Emoji.js');
 const { ActivityFlagsBitField } = require('../util/ActivityFlagsBitField.js');
 const { flatten } = require('../util/Util.js');
+const { Base } = require('./Base.js');
+const { Emoji } = require('./Emoji.js');
 
 /**
  * Activity sent in a message.
+ *
  * @typedef {Object} MessageActivity
  * @property {string} [partyId] Id of the party represented in activity
  * @property {MessageActivityType} type Type of activity sent
  */
 
 /**
  * The status of this presence:
- * * **`online`** - user is online
- * * **`idle`** - user is AFK
- * * **`offline`** - user is offline or invisible
- * * **`dnd`** - user is in Do Not Disturb
+ * - __`online`__ - user is online
+ * - __`idle`__ - user is AFK
+ * - __`offline`__ - user is offline or invisible
+ * - __`dnd`__ - user is in Do Not Disturb
```

Comment:
Let's just make this like `PresenceStatusData` (and everything else for that matter):

```suggestion
 * - `online` - user is online
 * - `idle` - user is AFK
 * - `offline` - user is offline or invisible
 * - `dnd` - user is in Do Not Disturb
```